### ISSUE TITLE
Webcam Multiface Facial Recognition

### ISSUE THREAD

**Issue by kampaitees** [WebClub-NITK/Hacktoberfest-2k19#59]
### Description
[Here](https://github.com/kampaitees/KSP-Internship-projects/blob/master/Criminal%20Recognition/Image%20Search%20from%20Database%20Method%201.ipynb) is the implemented version of Static Facial Recognition where input will be an image from the folder and output will be the similar images from the database. In this model, I assumed that there is only one face in the image. So I want you to convert this model to multi-face facial recognition and include the functionality of Webcam i.e. faces will be recognized for the webcam of your system(you can do the recognition of you and 2-3 your friends).

### Details
- Python, CNNs, Deep Learning, OpenCV, Computer Vision, Numpy, Dlib
- Type of issue: Single
- Time Limit: 2 Week
### Issue requirements / progress
- First, try to add the functionality of Webcam using OpenCV
- Then do the recognition of only one face at a time.
- Finally, include multi-face recognition into the pipeline

### Resources
[Model Weights](https://drive.google.com/drive/folders/1pwQ3H4aJ8a6yyJHZkTwtjcL4wYWQb7bn)
[68 Landmark Pedictor Weight](https://github.com/AKSHAYUBHAT/TensorFace/blob/master/openface/models/dlib/shape_predictor_68_face_landmarks.dat)
[CNN](https://towardsdatascience.com/a-comprehensive-guide-to-convolutional-neural-networks-the-eli5-way-3bd2b1164a53)
[Dlib Landmark Detector](http://dlib.net/face_landmark_detection.py.html)
[Dlib Facial Alignment](http://dlib.net/face_alignment.py.html)
[OpenCV](https://docs.opencv.org/4.1.1/d6/d00/tutorial_py_root.html)
[OpenCV Facial Recognition](https://github.com/kampaitees/OpenCV-projects/tree/master/Facial%20Recognition)
[Computer Vision](https://machinelearningmastery.com/what-is-computer-vision/)
[Deep Learning](https://en.wikipedia.org/wiki/Deep_learning)
[deeplearning.ai](https://www.youtube.com/watch?v=ArPaAX_PhIs&list=PLkDaE6sCZn6Gl29AoE31iwdVwSG-KnDzF)

### Directory Structure
Create a new folder called "59/Webcam Facial Recognition" inside the machine folder and add all the code corresponding to this issue inside /machine learning/59/Webcam Facial Recognition.

#### Note
Please claim the issue first by commenting here before starting to work on it.

#### For co-ordinators/maintainers creating the issues
For any further issues, feel free to contact me @kampaitees any time. Requests for review are welcomed. Happy Learning!

**Comment by SaurabhAgarwala** [comment]
Hacktoberfest 2019 ends. Closing the issue.
